zio/csvio/ztests/float.yaml

```diff
@@ -0,0 +1,22 @@
+zql: '*'
+
+input: |
+  #0:record[float64:float64]
+  0:[-;]
+  0:[0.0;]
+  0:[1.797693134862315708145274237317043567981e+308;]
+  0:[1.797693134862315708145274237317043567981e-324;]
```

Comment:
You're testing tzng parsing / overflow handling here rather than the csv output.

Reply:
True, though my aim here is to fix the current CSV representation of these values, which we should change (because they're `179769313486231570000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000` and `0`)!